x/tokenfactory/types/keys.go

```diff
@@ -47,3 +48,15 @@ func GetCreatorPrefix(creator string) []byte {
 func GetCreatorsPrefix() []byte {
 	return []byte(strings.Join([]string{CreatorPrefixKey, ""}, KeySeparator))
 }
+
+// GetDenomFeeWhitelistPrefix returns the store prefix of addresses which are
+// whitelisted to avoid a denom creation fee
+func GetCreateDenomFeeWhitelistPrefix() []byte {
```

Comment:
do we need a function for this? can't we just use `CreateDenomFeeWhitelistKey` since this function doesn't apend anything specific (I believe we can do prefix iteration even without joining the separator)

Reply:
I added this for consistency with the rest of the file, particularly `GetCreatorsPrefix` which is a similar scenario which was already in tokenfactory beforehand